.github/workflows/update-types.yml

```diff
@@ -88,4 +95,39 @@ jobs:
             Update types from ${{ steps.get_swagger_gh_uri.outputs.gh_uri }} ${{ github.event.inputs.single_path && format('(single path: {0})', github.event.inputs.single_path) || '' }}
           branch: ${{ github.event.inputs.single_path && format('autogenerate-{0}', github.event.inputs.single_path) || 'autogenerate' }}
           push_options: '--force'
-          create_branch: true
\ No newline at end of file
+          create_branch: true
```

Comment:
We should probably define / clarify the process we expect to follow for making types available with this work.

This GH workflow is really used to generate "proposed" changes - they are not directly made live without human intervention (we submit a PR from autogenerate -> main, review it, and then manually build + publish the nuget package before types are "officially" available).

There's a lot of scope to simplify this process (and I 100% feel like we should), but I still feel like there may be a need for separation between "generating types" and "reviewing + publishing them".

Reply:
I defined and got approval for the process to follow in https://github.com/Azure/bicep/discussions/11882. I am implementing the process now so hopefully this resolves the matter.